FrameRate/FrameRate.cpp

```diff
@@ -59,6 +64,8 @@ namespace WPEFramework
         {
             FrameRate::_instance = this;
 
+	    InitializeIARM();
```

Comment:
Pls move this to Initialize(). You can refer to how this is done for other plugins. You would have to return a empty string() from Initialize() to indicate that initialization was successful.